src/robot/parsing/populators.py

```diff
@@ -83,7 +83,7 @@ def start_table(self, header):
         self._populator.populate()
         table = self._datafile.start_table(DataRow(header).all)
         self._populator = self._populators[table.type](table) \
-                if table is not None else NullPopulator()
+            if table is not None else NullPopulator()
```

Comment:
As I already commented, PEP-8 fixes in general belong to own PRs. In this particular case I also happen to like the original style better. Using four spaces for indentation when continuing a line is confusing.